java/doc/api.rst

```diff
@@ -114,129 +108,47 @@ as the type of an input parameter.
 ``RayList<T>``
 ^^^^^^^^^^^^^^
 
-This is a list of ``RayObject<T>``s, which inherits from ``List<T>`` in Java. It
+This is a list of ``RayObject<T>``\s, which inherits from ``List<T>`` in Java. It
```

Comment:
What does the `\` do? Why not just ``RayObject<T>``s?

Reply:
Github won't render this correctly without the back slash.

Reply:
![image](https://user-images.githubusercontent.com/2883335/43376360-f4b8abe6-93ec-11e8-9e3b-5fa9123cb72a.png)

Reply:
Oh interesting.